src/routes.js

```diff
@@ -1,9 +1,16 @@
 const { Router } = require("express");
+const UserController = require("./controllers/UserController");
 
 const routes = new Router();
 
-routes.get("/", (req, res) => {
+routes.get("/", (_req, res) => {
   res.json({ message: "Hello World" });
 });
 
+routes.post("/users", (req, res) => UserController.createUser(req, res));
+
+routes.put("/users/:id", (req, res) => UserController.updateUser(req, res));
+
+routes.delete("/users/:id", (req, res) => UserController.deleteUser(req, res));
```

Comment:
Como os métodos da controller possuem os mesmos parâmetros da função das rotas, não é necessário fazer a arrow function.

```suggestion
routes.put("/users/:id", UserController.updateUser);

routes.delete("/users/:id", UserController.deleteUser);
```